src/reducers.js

```diff
@@ -30,20 +29,20 @@ import {
 function bootstrap(
   state = {
     bootstrap: {},
-    isAuthenticated: null,
-    isFetching: false
+    isFetching: true,
+    startedFetch: false,
```

Comment:
I think this is wrong. The initial `isFetching` state should be `false` instead of `true`. It only becomes true when the api request is sent to the backend.